pkg/scheduler/schedule_one.go

```diff
@@ -833,7 +834,9 @@ func getAttemptsLabel(p *framework.QueuedPodInfo) string {
 
 // handleSchedulingFailure records an event for the pod that indicates the
 // pod has failed to schedule. Also, update the pod condition and nominated node name if set.
-func (sched *Scheduler) handleSchedulingFailure(ctx context.Context, fwk framework.Framework, podInfo *framework.QueuedPodInfo, err error, reason string, nominatingInfo *framework.NominatingInfo, start time.Time) {
+func (sched *Scheduler) handleSchedulingFailure(ctx context.Context, fwk framework.Framework, podInfo *framework.QueuedPodInfo, errStatus *framework.Status, nominatingInfo *framework.NominatingInfo, start time.Time) {
+	err := errStatus.AsError()
```

Comment:
No need to call err.Error() again in L850, call `errMsg = status.Messages()` directly.